src/xcode/ENA/ENA/Source/Services/Exposure Submission/ExposureSubmissionService.swift

```diff
@@ -229,11 +230,13 @@ class ENAExposureSubmissionService: ExposureSubmissionService {
 					visitedCountries: self.supportedCountries,
 					checkins: unencryptedCheckins,
 					checkInProtectedReports: checkinProtectedReports,
-					completion: { result in
+					completion: { [weak self] result in
+						guard let self = self else {
```

Comment:
In case of pipeline Swift 5.7 compatibility:

```swift
guard let self else { ... }
```

Reply:
No it's not.